src/server/protocol.ts

```diff
@@ -681,6 +681,7 @@ namespace ts.server.protocol {
 
     export interface OrganizeImportsRequestArgs {
         scope: OrganizeImportsScope;
+        skipDestructiveCodeActions?: boolean;

```

Comment:
I don't think we have an actual convention, but it looks like we have a pattern of using positive names and, if necessary, treating undefined as true.  e.g. `UserPreferences.includeAutomaticOptionalChainCompletions` or `CompilerOptions.AllowUnreachableCode`.  I think @andrewbranch has also found this frustrating and might have suggestions?

Reply:
No, I don’t really have a preference here. This looks fine to me.